Applications/Games/Heroes of Might & Magic IV/Local (Gold Edition)/script.js

```diff
@@ -0,0 +1,47 @@
+const LocalInstallerScript = include("engines.wine.quick_script.local_installer_script");
+const Resource = include("utils.functions.net.resource");
+const { getLatestDevelopmentVersion } = include("engines.wine.engine.versions");
+const Downloader = include("utils.functions.net.download");
+
+new LocalInstallerScript()
+    .name("Heroes of Might & Magic IV")
+    .editor("New World Computing, Inc.")
+    .author("Zemogiter")
+    .category("Games")
+    .wineDistribution("upstream")
+    .wineVersion(getLatestDevelopmentVersion)
+    .executable("H4mod.exe")
+    .postInstall((wine) => {
+        const dataDirectory = wine.prefixDirectory() + "drive_c/" + wine.programFiles() + "Ubisoft/Heroes of Might & Magic IV - Złota Edycja/Data";
```

Comment:
Is this folder really correct? `Heroes of Might & Magic IV - Złota Edycja` looks to me like a localized (translated) string and nothing a publisher would normally use

Reply:
This is a default install location of my cd disc. I've set it in GOG local installer and it is correct.

Reply:
@madoar is correct. This will very likely not work in other countries. Maybe you can solve it with a regular expression. So: `ls` for the "Ubisoft" folder and then find the "Heroes of Might & Magic IV" folder.
Could even be enough to use [startsWith](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith).

Reply:
@plata the "Ubisoft" part is only there when you install from the disc. GOG installer uses "GOG.com" instead

Reply:
Shouldn't we have different installers then?